src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -180,31 +185,41 @@
     sendProductEvent(event, category, properties);
 
     if (COMPLETED_ORDER_PATTERN.matcher(event).matches()) {
+
+      // initialize instance of ScreenViewBuilder class
+      ScreenViewHitBuilder builder = new ScreenViewHitBuilder();
```

Comment:
FWIW this does seem weird to me that `ScreenViewBuilder` is being used for this. Can you link to the docs?